up & going/ch3.md

```diff
@@ -1,15 +1,15 @@
-# Usted no sabe JS: En marcha
-# Chapter 3: Dentro de YDKJS
+# You Don't Know JS: Up & Going
```

Comment:
@mballeng91 we need to take a decision about if we are going to translate this title (and the rest of them)

Reply:
Sure, I beilieve is better to let titles as the original version. Thanks @Zenfeuer

Reply:
Ok @mballeng91